yastd/dbus_api.md

```diff
@@ -0,0 +1,29 @@
+
+## get the list of available languages
+
+Iface: o.o.YaST.Installer1.Language
+
+stupid api:
+  AvailableLanguages -> array(string)
+
+minimal working api:
+  needed: human readable name (how localized?), identifier
+
+identifiers: maybe LanguageTag https://www.rubydoc.info/github/yast/yast-packager/master/LanguageTag
+- move it to yast-yast2
+- link to the standard from yard 
+- see https://tools.ietf.org/html/rfc4647 Matching of Language Tags
+- see https://lists.opensuse.org/archives/list/yast-devel@lists.opensuse.org/message/D52PSZ7TRID2RVM6CE6K2C2RUNNGOS6Z/
+
+return an array of objects (object paths) representing the languages?
+
+(later: keyboard layouts; territories? timezones? console fonts?)
```

Comment:
if you plan to extend it for timezone or keyboard layouts, then I think naming interface language is not propriete.

Reply:
Right, they should get their own interfaces. I was just looking at what yast2-country is doing.